modulefiles/fit2obs_hera.intel.lua

```diff
@@ -2,7 +2,8 @@ help([[
 Build environment for fit2obs on Hera
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+--prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
```

Comment:
Why leave the old `MODULEPATH` here?

Reply:
good catch.  Will remove once testing confirms this is good.